### PR TITLE
On Tools page Deleting `Settings History` freezes vertical scrolling

### PR DESCRIPTION
**Pull request by mi-prakash**
On Tools page Deleting `Settings History` freezes vertical scrolling

Task Url
https://ollyo.atlassian.net/browse/TUTOR-1749

### CODE REVIEW

File: assets/react/admin-dashboard/segments/import-export.js [hunk 1]
```diff
@@ -333,14 +337,14 @@ const delete_settings_xhttp_request = (modelOpener, modalElement) => {
...
-			// console.log(JSON.parse(xhttp.response));
 			modalElement.classList.remove('tutor-is-active');
 			let historyData = JSON.parse(xhttp.response);
 			historyData = historyData.data;
 			tutor_option_history_load(Object.entries(historyData));
 			delete_history_data();
 
-			setTimeout(function() {
+			setTimeout(function () {
+				// document.body.style.overflow = 'auto';
```
Comment (harunollyo) [inline assets/react/admin-dashboard/segments/import-export.js:347]: Uncomment this line

File: assets/react/admin-dashboard/segments/import-export.js [hunk 2]
```diff
@@ -64,6 +64,10 @@ const load_saved_data = () => {
 	};
 };
 
+function capitalizeFirstLetter(string){
+	return string.charAt(0).toUpperCase() + string.slice(1);
```
Comment (harunollyo) [inline assets/react/admin-dashboard/segments/import-export.js:68]: Need to `null` check

Comment (mi-prakash) [reply assets/react/admin-dashboard/segments/import-export.js:71]: added

File: assets/react/admin-dashboard/segments/import-export.js (hunk 1 above)
Comment (mi-prakash) [reply assets/react/admin-dashboard/segments/import-export.js]: done

**Review by harunollyo — APPROVED** [review]

**Pull request merged**